test-case/check-alsabat.sh

```diff
@@ -81,6 +81,37 @@ then
 	exit 2
 fi
 
+# Check device for AudioPlug Loopback enabled or not for LNL HDA
+if [ "$AUDIOPLUG_LOOPBACK" == "true" ] && [ "$MODEL" == "LNLM_RVP_HDA" ]; then
+
+	dlogi "The Device have AUDIO PLUG LOOPBACK enabled for LNLM_RVP_HDA"
+
+	# Check for playback/capture used USB codec, if yes replace with headset device
+        if [ "$pcm_c" == "hw:CODEC,0" ]; then
+	    pcm_c="hw:sofhdadsp,0"
+	    dlogi "Capture device changed to $pcm_c"
+        else
+	    pcm_p="hw:sofhdadsp,0"
+	    dlogi "Playback device changed to $pcm_p"
+	fi
+fi
+
+# Check device for AudioPlug Loopback enabled or not for LNL SDW
+if [ "$AUDIOPLUG_LOOPBACK" == "true" ] && [ "$MODEL" == "LNLM_SDW_AIOC" ]; then
```

Comment:
you probably want to avoid using the MODEL, we can use this test on previous generations as well.